### PR TITLE
Adding Dynatrace example to Fluent-Bit examples

### PR DESCRIPTION
**Pull request by aripolavarapu**
Worked with Mikhail Shapirov and Rodrigo Orzari at Amazon to review this contribution. 

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by shapirov103** [comment]
@PettitWesley do you mind taking a look at it, or advising whom to engage?

File: examples/fluent-bit/dynatrace/task-definition.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "family": "firelens-example-dynatrace",
+    "taskRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_iam_role",
+    "executionRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_execution_role",
+    "containerDefinitions": [
+        {
+            "essential": true,
+            "image": "amazon/aws-for-fluent-bit:stable",
```
Comment (PettitWesley) [inline examples/fluent-bit/dynatrace/task-definition.json:8]: Would you mind updating to our public ECR image: https://github.com/aws/aws-for-fluent-bit#amazon-ecr-public-gallery

```
public.ecr.aws/aws-observability/aws-for-fluent-bit:stable
```

For AWS customers, they can pull these images without any pull limits, whereas, this is not the case with Docker Hub.

Comment (PettitWesley) [inline examples/fluent-bit/dynatrace/task-definition.json]: Would you mind updating to our public ECR image: https://github.com/aws/aws-for-fluent-bit#amazon-ecr-public-gallery

```
public.ecr.aws/aws-observability/aws-for-fluent-bit:stable
```

For AWS customers, they can pull these images without any pull limits, whereas, this is not the case with Docker Hub.

**Review by PettitWesley — CHANGES_REQUESTED** [review]
@aripolavarapu Can you please make one change, then I am happy to merge this.

File: examples/fluent-bit/dynatrace/task-definition.json (hunk 1 above)
Comment (aripolavarapu) [reply examples/fluent-bit/dynatrace/task-definition.json]: Thanks for taking a look, made the change!

**Review by PettitWesley — APPROVED** [review]

**Pull request merged**